pkg/policy/api/rule.go

```diff
@@ -561,24 +610,33 @@ const (
 	KafkaMaxTopicLen = 255
 )
 
-// KafkaMaxTopicVal is the maximum value of supported API Keys in KafkaAPIKeyMap
-// KafkaReverseAPIKeyMap
+// KafkaInvalidAPIKey is the placeholder for a invalid apiKey value -1
 const (
-	KafkaMaxAPIKeyVal = 33
+	KafkaInvalidAPIKey = -1
```

Comment:
Removing the declaration below allows to remove this

Reply:
done